src/Controls/tests/TestCases.HostApp/Issues/Issue20834.cs

```diff
@@ -0,0 +1,82 @@
+﻿namespace Maui.Controls.Sample.Issues;
+
+[Issue(IssueTracker.Github, 20834, "[Android] GraphicsView can also be visible outside the canvas", PlatformAffected.Android)]
+public class Issue20834 : TestContentPage
+{
+	readonly Issue20834_Drawable drawable = new();
+	float oldX, oldY;
+	StackLayout rootLayout;
+	Grid grid;
+	GraphicsView graphicsView;
+
+	protected override void Init()
+	{
+		rootLayout = new StackLayout();
+		grid = new Grid()
+		{
+			BackgroundColor = Color.FromArgb("#1a2033"),
+			HeightRequest = 300,
+			VerticalOptions = LayoutOptions.Center
+		};
+		graphicsView = new GraphicsView() { AutomationId = "graphicsView" };
+		Label label = new Label()
+		{
+			AutomationId = "DropTarget",
+			Text = "Drop the circle here",
+			HeightRequest = 100,
```

Comment:
Is it possible to give this label a negative margin or replace the container with a grid so that the label is on top of the edge of the canvas?

The reason for this is so that we can see that the image is drawn at all. Maybe some freak thing happens and the clip clips everything. Also, it will make sure that the clip is also correctly scaled with the density.

Reply:
I have added another test case by replacing the stack with Grid . Please let me know if you have any concern.